tests/lib/test-common.bash.in

```diff
@@ -92,3 +78,43 @@ function get_average(){
 function random_name() {
 	echo $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
 }
+
+function check_active_process() {
+	process=$1
+	if pgrep -f "$process" > /dev/null; then
+		return 1
+	else
+		return 0
+	fi
+}
+
+# This function checks if there are containers or
+# cc-shim and qemu-lite processes up, if found, they are
+# killed to start test with clean environment.
+# The function is intended to be used in the setup() of
+# the docker integration tests for cc-oci-runtime.
+function kill_processes_before_start() {
+	[ "$($DOCKER_EXE ps -q)" ] && $DOCKER_EXE kill $("$DOCKER_EXE ps -q")
```

Comment:
Wasn't this the same error that Julio ran into, docker ps showed containers running but by the time kill was called, the containers were no longer running. Maybe have a return true for this statement to take this into account.